src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.MonoCrossAOT.sfxproj

```diff
@@ -0,0 +1,54 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.SharedFramework.Sdk" />
+
+  <PropertyGroup>
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <PlatformPackageType>RuntimePack</PlatformPackageType>
+    <SharedFrameworkName>Microsoft.NETCore.App.MonoCrossAOT</SharedFrameworkName>
+    <OverridePackageId>Microsoft.NETCore.App.MonoCrossAOT.$(RuntimeIdentifier).targeting.$(TargetCrossRid)</OverridePackageId>
```

Comment:
This should be `Microsoft.NetCore.App.Runtime.AOT.{host-rid}.Cross.{target-rid}.{version}`